web/run_debug.sh

```diff
@@ -0,0 +1 @@
+FLASK_APP=router FLASK_ENV=development flask run
```

Comment:
```suggestion
#!/bin/sh
FLASK_APP=router FLASK_ENV=development flask run
```
И `+x` если не выставлено еще